test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -71,6 +71,7 @@
     ("aten::_foreach_div_", datetime.date(2021, 3, 2)),
     ("aten::_foreach_addcdiv", datetime.date(2021, 3, 2)),
     ("aten::_foreach_addcmul", datetime.date(2021, 3, 2)),
+    ("aten::mkl_linear", datetime.date(2021, 3, 2)),
```

Comment:
it should be  **mkldnn_linear**.

Reply:
Oops, Will update after all tests finished (just to rule out the failure we had earlier).